### PR TITLE
[0.63] Reverse order of AccessibilityView in TransferProperties().

### LINKED ISSUE #6706: Setting a borderRadius style on a View causes accessibilityLabel to not be read by narrator
## Environment
Run the following in your terminal and copy the results here.
1. `npx react-native --version`:
4.13.0

2. `npx react-native run-windows --info`:
System:
    OS: Windows 10 10.0.19042
    CPU: (8) x64 Intel(R) Core(TM) i5-10210U CPU @ 1.60GHz
    Memory: 2.64 GB / 15.80 GB
  Binaries:
    Node: 10.15.0 - C:\Program Files\nodejs\node.EXE
    npm: 6.13.6 - C:\Repos\pay.pidl.sdk\tests\apps\pidl-react-native-windows\node_modules\.bin\npm.CMD
  npmPackages:
    react: 16.13.1 => 16.13.1
    react-native: 0.63.2 => 0.63.2
    react-native-windows: 0.63.3 => 0.63.3
  Installed UWP SDKs:
    10.0.17763.0
    10.0.18362.0

What device(s) are you targeting? Choose any from Desktop, Xbox, Hololens)
- Target Device(s): Desktop, Xbox

Which version of Visual Studio are you using? Specify Visual Studio 2017 or Visual Studio 2019
- Visual Studio Version: 2019

Which build configuration are you running? Choose from Debug, DebugBundle, Release, ReleaseBundle 
- Build Configuration: Debug/Release

## Steps To Reproduce
Provide a detailed list of steps that reproduce the issue.

1. Create a View
2. Create a Text element inside the View with the text "Test Text"
2. Set `accessibilityLabel={"Test Text"}` on the View
3. Set `accessibilityRole={"button"}` on the View
4: Set `style={{ borderRadius: 8 }}` on the View

## Expected Results
With narrator on, View should be focusable.  When focused, narrator should read the accessibilityLabel out loud as "Test Text"

## Actual Results
The view is focusable, but accessibility focus instead focuses the entire app.  If accessibilityRole is not set on the View , the View becomes focusable by the narrator but is read as a custom role.  If borderRadius is removed, the element becomes focusable by the narrator but will the lack the desired border radius style.

### PR DESCRIPTION
**Pull request by rectified95**
In v0.63 setting `View.borderRadius` causes narrator to set its focus on the entire app windows, instead of the View in question. 

In `FrameworkElementViewManager::TransferProperties()`, if `oldView == newView`, then
```cpp
xaml::Automation::AutomationProperties::SetAccessibilityView(newView, accessibilityView);
xaml::Automation::AutomationProperties::SetAccessibilityView(oldView, winrt::Peers::AccessibilityView::Raw);
```
results in the value being overwritten incorrectly. Reversing these 2 operations fixes the problem, and doesn't change anything if `oldView != newView`.

Issue doesn't occur in 0.64, though, despite this code being identical.

Fixes #6706 

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/6847)

### CODE REVIEW

**Review by stecrain — APPROVED** [review]

**Comment by stecrain** [comment]
thanks for bringing this over!

**Review by marlenecota — APPROVED** [review]

**Comment by rectified95** [comment]
Adding reviewers with stable branch privileges.

**Comment by NickGerleman** [comment]
Could someone on this thread help me understand why this applies to 0.63, but not 0.64? Curious what the motivation was before for setting AccessibilityView on the old view before.

**Comment by rectified95** [comment]
I simply wasn't able to reproduce the issue on 0.64, so didn't investigate further. It does repro on 0.63, though. Fix itself comes from @stecrain
Should I create a PR for 0.64 as well anyway?

**Comment by NickGerleman** [comment]
@rectified95 I think it would be valuable to debug a bit to understand more. If we think the order was incorrect in the first place, there might be a separate bug in newer branches for example.

E.g. is there a difference in the XAML tree in 0.64 vs 0.63 before the change? Do we hit the same code path edited here?

**Comment by rectified95** [comment]
@NickGerleman @asklar I think I've figured out why this doesn't happen in 0.64: 
in both releases the call to `TransferProperties()` is skipped depending on whether a border needs to be added/removed. That determination depends on the information about the outer border stored in the panel wrapping a View; this state is updated by calls to `ViewPanel::FinalizeProperties` which looks up whether the shadow node stores info about a corner radius.

Now, in 0.63 we call `FinalizeProperties()` in every call to `ViewViewManager::UpdateProperty` in a loop inside `ViewViewManager::UpdateProperites`, while in 0.64 it's done only after all props have been updated in `OnUpdatedProperties`, which doesn't exist in 0.63. 

To sum up, when the View is first created, `oldView!= newView`, and calling `FinalizeProperties` once is fine. However, when `FinalizeProperties` is called several times, each time in `UpdateProperty`, then `oldView == newView`, which results in the value of `AccessibilityView` being overwritten,

**Comment by NickGerleman** [comment]
Thanks for taking look. It sounds like we fixed the root cause of the issue in 0.64 then. Approving this change to go back to 0.63 since folks have asked for it.

**Review by NickGerleman — APPROVED** [review]
